src/components/Employees/index.tsx

```diff
@@ -0,0 +1,214 @@
+import styles from './Employees.module.css';
+import employees from '@components/Employees/employeesData';
+import EmployeeItem from '@components/EmployeeItem';
+import { IEmployeeItem } from '@components/EmployeeItem/types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+import { useSearchParams } from 'react-router-dom';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+function Employees() {
+    const { t } = useTranslation();
+    const [searchParams, setSearchParams] = useSearchParams({});
+    const [searchActiveButtons, setSearchActiveButtons] = useState<string[]>(
+        []
+    );
+
+    function searchButtonExists(searchParam: string): boolean {
+        return searchActiveButtons.some(
+            (searchBtn) => searchBtn === searchParam
+        );
+    }
+
+    function employeeExists(
+        searchParam: IEmployeeItem,
+        filteredEmployees: IEmployeeItem[]
+    ): boolean {
+        return filteredEmployees.some((searchBtn) => searchBtn === searchParam);
+    }
+
+    function addFilter(target: string, keyParam: string) {
```

Comment:
You can rename the `keyParam` to just key and target to value, as it is doesn't represent the target (event.target) it's event.target.value